app/src/pages/AlgorithmsPage/AlgorithmsPage.tsx

```diff
@@ -7,9 +7,12 @@ import { connect } from 'react-redux';
 import { getAlgorithms } from 'store/actions/AlgorithmsStoreActions';
 import { SnowmanDispatch } from 'store/messages';
 import { Store } from 'store/models';
+import { FirstValidMatchingSolution } from 'utils/constants';
 
 const mapStateToProps = (state: Store): AlgorithmsPageStateProps => ({
-  algorithms: state.AlgorithmsStore.algorithms,
+  algorithms: state.AlgorithmsStore.algorithms.filter(
+    (value) => value.id >= FirstValidMatchingSolution
```

Comment:
It should be (anAlgorithm: Algorithm) => anAlgorithm.id >= FirstValidMatchingSolution

Reply:
Will be resolved in separate commit :)